packages/cozy-scanner/babel-transformer.js

```diff
@@ -0,0 +1,3 @@
+const config = require('./babel.config')
+const { createTransformer } = require('babel-jest')
+module.exports = createTransformer(config)
```

Comment:
Why is this necessary ? Can you comment ? Is this because we use a custom babel config ?

Reply:
Comment added: 

/**
 * Since we are using a custom babel config
 * we need to create a Transformer to tell
 * jest to use it
 */